torch/_export/verifier.py

```diff
@@ -117,48 +130,13 @@ def check_valid_aten_dialect(gm: GraphModule) -> None:
                     )
                 )
 
-    # Tensors be of contiguous format
-    for name, param in itertools.chain(gm.named_parameters(), gm.named_buffers()):
-        if isinstance(param, torch.Tensor):
-            if not param.is_contiguous():
-                raise SpecViolationError(
-                    f"Tensors in Aten dialect must be contiguous, {name} is not contiguous"
-                )
+    _check_tensors_are_contiguous(gm)
 
 
-@torch.fx._compatibility.compatibility(is_backward_compatible=False)
+@compatibility(is_backward_compatible=False)
 def is_valid_aten_dialect(gm: GraphModule) -> bool:
     try:
         check_valid_aten_dialect(gm)
         return True
     except SpecViolationError:
         return False
-
-
-@torch.fx._compatibility.compatibility(is_backward_compatible=False)
-def check_valid_edge_dialect(gm: GraphModule) -> None:
```

Comment:
We may not need edge dialect validation for export, but we will need it in the following compilation stage. Do we want to keep it somewhere?

Reply:
Yeah, these checks still exist in the internal codebase.